core/Services/Calendar/Util/ICalTime.vala

```diff
@@ -62,22 +64,25 @@ namespace Calendar.Util {
                 return new GLib.TimeZone (tzid);
             }
         }
-
-        //FIXME See https://github.com/libical/libical/pull/513
         // If tzid fails, try ICal.Time.get_timezone ()
-        ICal.Timezone* timezone = date.get_timezone ();
+        unowned ICal.Timezone? timezone = null;
+        if (timezone == null && date.get_timezone () != null) {
```

Comment:
Why test for `null` here when the previous line set it to `null`?

Reply:
Hmm, I did a blind reverse of the commit but it fixed some things too